skyeye-config-example.yaml

```diff
@@ -0,0 +1,10 @@
+log-level: info
+log-format: json
+telemetry-address: "localhost:42674"
+srs-server-address: "localhost:5002"
+callsign: "Magic1"
```

Comment:
I actually haven't tested if GCI callsigns with numbers in them work 😅 set this to "Magic", since that does have test coverage

Reply:
Done